### PR TITLE
feat(gatsby-plugin-less): add support for overriding the default options of `css-loader`

### LINKED ISSUE #9142: [Gatsby-plugin-less]Support override the default options of css loader
## Summary

Gatsby-plugin-less support override the default options of css loader. Such as `localIdentName` and `camelCase`.

### Basic example

in `gatsby-config.js`:
```
module.exports = {
  plugins: [
    {
      resolve: 'gatsby-plugin-less',
      options: {
        overrideCssLoaderOpts: {
           camelCase: false,
           // ... other css loader configuration
        }
      },
    },
  ],
}
```
### Motivation

The default css loader options provided by Gatsby will set camelCase is `true`.
But the third-party lib we reference does not use `camelCase` and we need to reference its `less` file.

For example, use `antd` and `babel-plugin-import` same time.

### PR DESCRIPTION
**Pull request by yeskunall**
Yay! 💖

<hr />
Closes #9142.

### CODE REVIEW

**Comment by pieh** [comment]
Some docs updated in README of the package would be great

**Comment by pieh** [comment]
Code itself looks good to me! :)

File: packages/gatsby-plugin-less/src/gatsby-node.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ exports.onCreateWebpackConfig = (
       ? [loaders.null()]
       : [
           loaders.miniCssExtract(),
-          loaders.css({ importLoaders: 2 }),
+          loaders.css({ ...cssLoaderOptions, importLoaders: 2 }),
```
Comment (yeskunall) [inline packages/gatsby-plugin-less/src/gatsby-node.js]: @pieh I was gonna update the docs, but I noticed that the feature request was made for `css-loader` and as I understand, you can chain `css-loader` with `less-loader`. I haven't looked into this much, but I am curious as to how this works.

I feel like if I understand this it'll help me update the docs appropriately. Thanks for understanding 😅

Comment (yeskunall) [reply packages/gatsby-plugin-less/src/gatsby-node.js]: Holy! That's an in-depth explanation. Still reading it, but just wanted to say I really appreciate it! 🤯

File: packages/gatsby-plugin-less/README.md [hunk 2]
```diff
@@ -31,6 +31,20 @@ plugins: [
...
 ```
 
+If you need to override the default options passed into [`css-loader`](https://github.com/webpack-contrib/css-loader)
+
+```javascript
+// in gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-plugin-less`,
+    cssLoaderOptions: {
```
Comment (pieh) [inline packages/gatsby-plugin-less/README.md]: This need to be inside `options` field:
```
plugins: [
  {
    resolve: `gatsby-plugin-less`,
    options: {
      cssLoaderOptions: {
        camelCase: false,
      },
    },
  }
]

**Comment by yeskunall** [comment]
Thanks for being patient @pieh. I think it's okay to merge now 😅sorry for all the trouble!

Thanks 💖

**Comment by pieh** [comment]
> Thanks for being patient @pieh. I think it's okay to merge now 😅sorry for all the trouble!

Don't worry about it - it's all good :)
Now we just need to wait for github to start working correctly again :)

**Pull request merged**